client/app/bundles/course/experience-points/disbursement/components/tables/ForumDisbursementTable.tsx

```diff
@@ -0,0 +1,238 @@
+    },
+    {
+      name: intl.formatMessage(translations.postCount),
+      label: intl.formatMessage(translations.postCount),
+      options: {
+        filter: false,
+        sort: false,
+        setCellHeaderProps: () => ({
+          style: { padding: '14px', textAlign: 'end' },
+        }),
+        setCellProps: () => ({
+          style: {
+            overflowWrap: 'anywhere',
+            padding: '5px 14px',
+            textAlign: 'end',
+          },
+        }),
+        customBodyRenderLite: (dataIndex: number): JSX.Element => (
+          <>
+            {dataIndex === 0 && (
+              <ReactTooltip id="view-posts">
+                <FormattedMessage {...translations.viewPosts} />
+              </ReactTooltip>
+            )}
+            <Link
+              component="button"
+              className={`view-posts-${forumUsers[dataIndex].id}`}
+              form=""
+              onClick={(): void => {
+                onPostClick(
+                  forumUsers[dataIndex].name,
+                  forumUsers[dataIndex].id,
+                );
+              }}
+              data-tip
+              data-for="view-posts"
+            >
+              {forumUsers[dataIndex].postCount}
+            </Link>
+          </>
+        ),
+      },
+    },
+    {
+      name: intl.formatMessage(translations.voteTally),
+      label: intl.formatMessage(translations.voteTally),
+      options: {
+        filter: false,
+        sort: false,
+        setCellHeaderProps: () => ({
+          style: { padding: '14px', textAlign: 'end' },
+        }),
+        setCellProps: () => ({
+          style: {
+            overflowWrap: 'anywhere',
+            padding: '5px 14px',
+            textAlign: 'end',
+          },
+        }),
+        customBodyRenderLite: (dataIndex: number): string =>
+          forumUsers[dataIndex].voteTally.toString(),
+      },
+    },
+    {
+      name: intl.formatMessage(translations.pointsAwarded),
+      label: intl.formatMessage(translations.pointsAwarded),
+      options: {
+        filter: false,
+        sort: false,
+        setCellHeaderProps: () => ({
+          style: { padding: '14px' },
+        }),
+        setCellProps: (_, rowIndex: number) => ({
+          className: 'course_user',
+          id: `course_user_${forumUsers[rowIndex].id}`,
+          style: { overflowWrap: 'anywhere', padding: '5px 14px' },
+        }),
+        customBodyRenderLite: (dataIndex: number): JSX.Element =>
+          renderFilteredPointTextFields[dataIndex],
+      },
+    },
+  ];
+
+  const options: TableOptions = {
+    download: false,
+    filter: false,
+    pagination: false,
+    print: false,
+    search: false,
+    selectableRows: 'none',
+    selectToolbarPlacement: 'none',
+    viewColumns: false,
+  };
+
+  return (
+    <DataTable
+      data={forumUsers}
+      options={options}
+      columns={columns}
+      height="10px"
+    />
+  );
+};
+
+export default injectIntl(ForumDisbursementTable);
```

Comment:
add memo here and compare the props. can use equal also

Reply:
I have updated with memo, but I also realised that the field array need to be re-rendered to show the values correctly e.g. when copying the first row EXP to all the rows, so I added it into the comparison.